apps/grav/index.js

```diff
@@ -13,9 +13,12 @@ console.log('grav-theme ran');
 
 // Send each component the $(document) as its context
 const $context = $(document);
+const settings = {
+  enableHolder: TRUE,
+};
 
 // Let's just execute everything and pass in $(document), settings
 _.forEach(designSystem, (component) => {
```

Comment:
@jryanconklin we've got a MUCH more elegant way to do this now, check the drupal/pl index.js files for deets (we now have api methods like `enableAllComponents` on the design-system library!).